crd-diff-config.yaml

```diff
@@ -0,0 +1,109 @@
+checks:
+  crd:
+    scope:
+      enabled: true
+    existingFieldRemoval:
+      enabled: true
+    storedVersionRemoval:
+      enabled: true
+  version:
+    sameVersion:
+      enabled: true
+      unhandledFailureMode: "Closed"
+      enum:
+        enabled: true
+        removalEnforcement: "Strict"
+        additionEnforcement: "Strict"
```

Comment:
This enforcement means addition of new allowed enum values is considered incompatible. I know we have some areas of our API where we anticipate adding some enum values so we may want to move this to the less strict `IfPreviouslyConstrained` that allows addition of new allowed enum values only if the property was previously constrained by enum values. (i.e adding net new enum restrictions to a property is still considered incompatible)